### PR TITLE
SQL-1359: Download center release feed

### PR DESCRIPTION
**Pull request by nbagnard**

### CODE REVIEW

File: evergreen.yml [hunk 1]
```diff
@@ -301,6 +303,28 @@ functions:
...
+          sed -i 's@{RELEASE_PQX_PATH}@'${RELEASE_PQX_PATH}'@' mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+          cat mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+    - command: s3.put
+      params:
+        aws_key: ${release_aws_key}
+        aws_secret: ${release_aws_secret}
+        local_file: mongo-tableau-connector/mongo-tableau-connector-downloads_template.json
+        remote_file: mongo-tableau-connector/mongo-tableau-connector-downloads.json
```
Comment (terakilobyte) [inline evergreen.yml:323]: It looks like there might be some copypasta.
```suggestion
        local_file: mongo-powerbi-connector/mongo-powerbi-connector-downloads_template.json
        remote_file: mongo-powerbi-connector/mongo-powerbi-connector-downloads.json
```

File: evergreen.yml [hunk 2]
```diff
@@ -301,6 +303,28 @@ functions:
...
+        bucket: translators-connectors-releases
+        permissions: public-read
+        content_type: application/json
```
Comment (terakilobyte) [inline evergreen.yml:326]: Should this be `application/json`? If it is signifying the download type for the pqx file, it should be [`application/octet-stream`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)

**Review by terakilobyte — CHANGES_REQUESTED** [review]
Just a few minor things I noticed.

File: evergreen.yml (hunk 1 above)
Comment (nbagnard) [reply evergreen.yml]: 😩 Yes, copy-paste, copy-paste

File: evergreen.yml (hunk 2 above)
Comment (nbagnard) [reply evergreen.yml:326]: No, it means that the file `mongo-powerbi-connector-downloads.json` is a json file

File: evergreen.yml (hunk 1 above)
Comment (terakilobyte) [reply evergreen.yml]: Happens to us all!

File: evergreen.yml [hunk 3]
```diff
@@ -301,6 +303,28 @@ functions:
...
+          sed -i 's@{RELEASE_PQX_PATH}@'${RELEASE_PQX_PATH}'@' mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+          cat mongo-powerbi-connector-downloads_template.json
+          echo "-------------------------"
+    - command: s3.put
+      params:
+        aws_key: ${release_aws_key}
+        aws_secret: ${release_aws_secret}
+        local_file: mongo-tableau-connector/mongo-powerbi-connector-downloads_template.json
+        remote_file: mongo-tableau-connector/mongo-powerbi-connector-downloads.json
```
Comment (terakilobyte) [inline evergreen.yml:323]: I'm not sure if all the changes propagated, but I'm still seeing two instances of `tableau`.

```suggestion
        local_file: mongo-powerbi-connector/mongo-powerbi-connector-downloads_template.json
        remote_file: mongo-powerbi-connector/mongo-powerbi-connector-downloads.json
```

**Review by terakilobyte — APPROVED** [review]
I'm still seeing two instances of `tableau` that I think need changed. Other than that, LGTM. No need to retag for review.

**Review by bucaojit — APPROVED** [review]
LGTM

**Comment by nbagnard** [comment]
evergreen merge

**Pull request merged**